### PR TITLE
Add monthly update template

### PR DESCRIPTION
**Pull request by majetideepak**

### CODE REVIEW

File: velox/docs/README.md [hunk 1]
```diff
@@ -45,3 +45,34 @@ git commit -m "Update documentation"
...
+- Presto Functions
+- Spark Functions
+- Hive Connector
+- Performance and Correctness
+- Build System
+- Credits
+
+Below are a few monthly update guidelines.
+- Each update should be in the imperative, present tense similar to a git commit.
+- In sections where applicable, start with the new additions with `Add` followed by
```
Comment (kgpai) [inline velox/docs/README.md:74]: nit: Maybe also specify to, Start with 'Adds' followed by Fixes , followed by other .

File: velox/docs/README.md [hunk 2]
```diff
@@ -45,3 +45,36 @@ git commit -m "Update documentation"
...
+- Performance and Correctness
+- Build System
+- Credits
+
+Below are a few monthly update guidelines.
+- Each update should be in the imperative, present tense similar to a git commit.
+- In sections where applicable, start new additions with `Add` followed by
+  the fixes using `Fix`, followed by other updates.
+- Group similar contributions together. Example: All Presto scalar functions can be
+  combined in one line.
```
Comment (pedroerp) [inline velox/docs/README.md:79]: Nice! Maybe we could add a pointer to where these files should be added, and/or a link to the directory?

**Comment by pedroerp** [comment]
@majetideepak is this still in draft mode?

**Comment by majetideepak** [comment]
@pedroerp I addressed your review comment. This is ready.

**Review by pedroerp — APPROVED** [review]
Thanks @majetideepak

**Pull request closed without merging**